hadoop-hdds/docs/themes/ozonedoc/layouts/partials/navbar.html

```diff
@@ -34,7 +34,7 @@
     <div id="navbar" class="navbar-collapse collapse">
       <ul class="nav navbar-nav navbar-right">
         <li><a href="https://github.com/apache/hadoop-ozone">Source</a></li>
```

Comment:
Should change to 
https://github.com/apache/ozone

Reply:
Thanks to @ayushtkn  for the review.
Of course, let me confirm what you mentioned.

Reply:
@ayushtkn I searched the README.md under Ozone, there should be no other errors